_sass/module/_ballot-nav.scss

```diff
@@ -6,51 +6,67 @@ $ballot-nav-border: 2px $color-grey-4-5 solid;
     padding-right: $gutter--large;
   }
 
-  h1,
-  h2,
-  h3,
-  h4,
-  h5,
-  h6 {
-    color: $heading-color--light;
-    font-weight: $font-weight-light;
-  }
```

Comment:
Ah, were these duplicated/unnecessary because of the stuff in `base/_typography.scss`?

Reply:
It's not that they were unnecessary. Because there were nested w/i .ballot-nav (so the selector was, for example, `.ballot-nav h1`), it was making it hard to apply different styles to the different headings within the ballot-nav. I unnested these styles and split them out into specific classes for the different kinds of headings in the ballot-nav.